services/pdf-ms/lambdas/addPdfToCase.ts

```diff
@@ -17,12 +17,14 @@ import {
   getNewAndChangedCaseAnswerValues,
 } from '../helpers/case';
 
+import log from '../../../libs/logs';
+
 const PDF_SSM_PARAMS = params.read(config.pdf.envsKeyName);
 
 // Convert DynamoDB item to plain object
 const dynamoDbConverter = AWS.DynamoDB.Converter;
 
-export async function main(event: Record<string, any>): Promise<Boolean> {
+export async function main(event: Record<string, any>, context): Promise<Boolean> {
```

Comment:
To solve this code issue you could add **Record<string, any>** after context.

Reply:
Added.